lib/binding_web/tree-sitter-web.d.ts

```diff
@@ -1,12 +1,15 @@
 declare module 'web-tree-sitter' {
   class Parser {
     static init(): Promise<void>;
-    delete(): void;
-    parse(input: string | Parser.Input, previousTree?: Parser.Tree, options?: Parser.Options): Parser.Tree;
-    getLanguage(): any;
-    setLanguage(language: any): void;
-    getLogger(): Parser.Logger;
-    setLogger(logFunc: Parser.Logger): void;
+    delete();
+    parse(input: string | Parser.Input, previousTree?: Parser.Tree, options?: Parser.Options): Parser.Tree | null;
+    getLanguage(): Parser.Language | null;
+    setLanguage(language?: Parser.Language): void;
+    getLogger(): Parser.Logger | null;
+    setLogger(logFunc?: Parser.Logger): void;
+    reset(): void;
```

Comment:
For functions with no return value, is the `void` useful or not? I noticed you removed some of the explicit `void` annotations, but not all.

Reply:
It doesn't make any semantic difference as far as I know. I think the IDE tooling or something might warn you about the explicit ones which is why I might have removed them. It's been awhile since I looked at this though, so I don't remember for sure now.